src/components/App/TopNav.js

```diff
@@ -79,11 +83,11 @@ export default function TopNav(props: TopNavProps) {
       <div className="ui input">
         <TraceIDSearchInput />
       </div>
-      {NAV_LINKS.map(({ key, to, text }) =>
+      {NAV_LINKS.map(({ key, to, text }) => (
```

Comment:
`husky > npm run -s precommit (node v6.11.2)`

does this change

Reply:
@pavolloffay I don't follow... can you elaborate?

Reply:
that this changes it done automatically via git hooks not by me